hieradata/common.yaml

```diff
@@ -259,6 +259,9 @@ govuk::apps::content_store::nagios_memory_critical: 1300
 govuk::apps::content_tagger::db_hostname: "postgresql-primary-1.backend"
 govuk::apps::content_tagger::db::backend_ip_range: "%{hiera('environment_ip_prefix')}.3.0/24"
 
+govuk::apps::content_tagger::redis_host: 'redis-1.backend'
```

Comment:
You don't need these two lines as the defaults in the puppet class will set them correctly.